### DISCUSSION TITLE
Does the package support GO v1.17.2

### DISCUSSION THREAD

**Discussion by gitHusband** [gavv/httpexpect#146] [Q&A]
I can not download the package using
`go get -v github.com/gavv/httpexpect/v2`

The error is
`go get: module github.com/gavv/httpexpect/v2: stream error: stream ID 1; INTERNAL_ERROR`

My Go Version:

`go version go1.17.2 darwin/amd64`

**Comment by gavv** [comment]
Hi. This looks like network issue, not related to go version and to the package itself.
